src/promptflow-evals/promptflow/evals/evaluators/coherence/flow/aggregate.py

```diff
@@ -0,0 +1,10 @@
+from typing import List
```

Comment:
Do we need aggregate logic ?

Reply:
We might need the aggregration logic for batch run.  Let's keep them in the flow for now.

Reply:
A decision was made for evaluators to not have aggregation logic. Evaluator should produce values for the single input. Aggregation will come in later but would be outside of evaluators (as of now)

Reply:
Aggregation logic removed.